build.gradle

```diff
@@ -204,6 +206,18 @@ dependencies {
     // library to build pdf reports
     compile 'com.github.dhorions:boxable'
 
+    //support for Oracle DB version 12.1.0.2
+    providedCompile group: 'com.oracle', name: 'ojdbc7', version: '12.1.0.2'
```

Comment:
why not testCompile ?

Reply:
changed to 'testCompile' and commented because Oracle jdbc dependency can be downloaded only with credentials. I just kept it there to mention the version used to test Oracle DB for future needs.